available-connectors.json

```diff
@@ -735,7 +735,7 @@
     },
     "supportedByStrongLoop": false
   },
-  {
+  "zos": {
```

Comment:
`zos` is an OS. This is too generic, please make it something like `zosconnectee` or `zos-connect-ee`.